bedrock/base/templates/includes/protocol/footer/footer.html

```diff
@@ -6,15 +6,18 @@
 
 {% set utm_params = 'utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer' %}
 
-<footer class="c-footer {% if theme_class %}{{ theme_class }}{% endif %}" id="colophon">
+<footer class="mzp-c-footer {% if theme_class %}{{ theme_class }}{% endif %}" id="colophon">
```

Comment:
There are a few places in JS that reference this component by name, so we'll need to update those references.

One place is the sticky promo script that prevents the promo from covering the language switcher:
https://github.com/mozilla/bedrock/blob/master/media/js/firefox/sticky-promo.js#L115

Another spot is `js/base/protocol/protocol-footer.js` but that's just a copy of the same script in Protocol. I think we can update the static bundle to use the Protocol version and delete the copy in bedrock:
https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json#L1598

Reply:
Thanks for the context and JS pointers! Will update as directed 👍